providers/slack/tests/unit/slack/hooks/test_slack_webhook.py

```diff
@@ -432,6 +432,7 @@ def test_hook_send_dict_legacy_slack_integration(self, mock_webhook_client_cls,
             {"text": "Test Text"},
             {"text": "Fallback Text", "blocks": ["Dummy Block"]},
             {"text": "Fallback Text", "blocks": ["Dummy Block"], "unfurl_media": True, "unfurl_links": True},
+            {"legacy": "value"},
```

Comment:
?  Needed change or not?

Reply:
I noticed that it supports other kwargs for legacy integration: https://github.com/apache/airflow/blob/85de1fe9b80e82db3f7b668bc23f56c66b1f75fc/providers/slack/src/airflow/providers/slack/hooks/slack_webhook.py#L246-L247
But that wasn't being tested in unit tests so I added this in